scripts/windows/setup/IotEdgeSecurityDaemon.ps1

```diff
@@ -252,7 +269,8 @@ function Initialize-IoTEdge {
         [String] $Username,
 
         # Password used to access the container registry and pull the IoT Edge Agent image.
-        [SecureString] $Password
+        [SecureString] $Password,
+        [switch] $Force=$false
```

Comment:
if $Force set to true, but IoT Edge is already installed, what happened? should it work?

Reply:
answered in previous comment above.